packages/common-all/src/types/seed.ts

```diff
@@ -6,6 +6,15 @@ export type SeedConfig = {
   description: string;
   repository: SeedRepository;
   contact?: SeedContact;
+  /**
+   * Url for seed
+   */
+  site?: SeedSite;
+};
+
+export type SeedSite = {
+  url: string;
+  index?: string;
```

Comment:
What does index here mean/do?

Reply:
equivalent of `siteIndex`, see https://wiki.dendron.so/notes/f2ed8639-a604-4a9d-b76c-41e205fb8713.html#siteindex-optional